cluster_test.go

```diff
@@ -0,0 +1,42 @@
+package test
+
+import (
+	"crypto/tls"
+	"fmt"
+	"testing"
+
+	. "github.com/onsi/gomega"
+
+	http_helper "github.com/gruntwork-io/terratest/modules/http-helper"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+)
+
+func TestCluster(t *testing.T) {
+	g := NewGomegaWithT(t)
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "./",
+	})
+
+	defer terraform.Destroy(t, terraformOptions)
+
+	terraform.InitAndApply(t, terraformOptions)
+
+	clusterOptions := k8s.NewKubectlOptions("", "./kubeconfig", "default")
```

Comment:
After TF creates a cluster, kubeconfig of the created cluster is written out to .kube/config  automatically?

Reply:
yes

https://github.com/civo/terraform-template/blob/2155e734c84c0f435888dbee82934a91d9163fde/cluster.tf#L11